docs/resources/svm_peers_resource.md

```diff
@@ -0,0 +1,136 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "ONTAP: SVM Peers"
+subcategory: "SVM Peers"
+description: |-
+  Svm resource
+---
+
+# Resource SVMPeers
+
+Create/Modify/Delete a SVMPeer
+
+### Related ONTAP commands
+* vserver peer create
+* vserver peer accept
+* vserver peer modify
+* vserver peer delete
+
+## Supported Platforms
+* On-perm ONTAP system 9.6 or higher
+
+~> **NOTE:** Amazon FSx for NetApp ONTAP is not supported
```

Comment:
you sure on this?

Reply:
Nope, shall I remove it?, I thought its for most of the resources for now